use-timescale/data-tiering/querying-tiered-data.md

```diff
@@ -11,11 +11,13 @@ tags: [ storage, data management ]
 <!-- vale Google.Acronyms = NO -->
 <!-- vale Google.Headings = NO -->
 
-Once rarely used data is tiered and migrated to low cost object storage it can still be queried with standard SQL by
-enabling the `timescaledb.enable_tiered_reads` GUC. 
+Once rarely used data is tiered and migrated to low cost object storage it can still be queried 
+with standard SQL by enabling the `timescaledb.enable_tiered_reads` GUC. 
```

Comment:
I think that we should add a note that querying is turned off by default. What's your opinion?

Reply:
Agree. I am not sure if that change has been deployed yet. Need to verify before updating the doc.